commit ec58163fb67c1456f8e01c54bc6d283dd5cf2849
Author: dpgilbert <dpgilbert>
Date:   Sat Apr 15 14:29:01 2006 +0000

    VPD page 0x83 size is larger in linux, increase to 252 bytes

diff --git a/scsiprint.c b/scsiprint.c
index 0b7c70b..4a1bbdb 100644
--- a/scsiprint.c
+++ b/scsiprint.c
@@ -41,7 +41,7 @@ #include "utility.h"
 
 #define GBUF_SIZE 65535
 
-const char* scsiprint_c_cvsid="$Id: scsiprint.c,v 1.107 2006/04/12 16:18:57 ballen4705 Exp $"
+const char* scsiprint_c_cvsid="$Id: scsiprint.c,v 1.108 2006/04/15 14:29:01 dpgilbert Exp $"
 CONFIG_H_CVSID EXTERN_H_CVSID INT64_H_CVSID SCSICMDS_H_CVSID SCSIPRINT_H_CVSID SMARTCTL_H_CVSID UTILITY_H_CVSID;
 
 // control block which points to external global control variables
@@ -881,7 +881,7 @@ static int scsiGetDriveInfo(int device, 
         /* <<<< This is Linux specific code to detect SATA disks using a
                 SCSI-ATA command translation layer. This may be generalized
                 later when the t10.org SAT project matures. >>>> */
-        req_len = 96;
+        req_len = 252;
         memset(gBuf, 0, req_len);
         if ((err = scsiInquiryVpd(device, 0x83, gBuf, req_len))) {
             PRINT_ON(con);
diff --git a/smartd.c b/smartd.c
index 63242fe..eb26578 100644
--- a/smartd.c
+++ b/smartd.c
@@ -115,14 +115,14 @@ #define ARGUSED(x) ((void)(x))
 extern const char *atacmdnames_c_cvsid, *atacmds_c_cvsid, *ataprint_c_cvsid, *escalade_c_cvsid, 
                   *knowndrives_c_cvsid, *os_XXXX_c_cvsid, *scsicmds_c_cvsid, *utility_c_cvsid;
 
-static const char *filenameandversion="$Id: smartd.c,v 1.362 2006/04/12 16:18:57 ballen4705 Exp $";
+static const char *filenameandversion="$Id: smartd.c,v 1.363 2006/04/15 14:29:25 dpgilbert Exp $";
 #ifdef NEED_SOLARIS_ATA_CODE
 extern const char *os_solaris_ata_s_cvsid;
 #endif
 #ifdef _WIN32
 extern const char *daemon_win32_c_cvsid, *hostname_win32_c_cvsid, *syslog_win32_c_cvsid;
 #endif
-const char *smartd_c_cvsid="$Id: smartd.c,v 1.362 2006/04/12 16:18:57 ballen4705 Exp $" 
+const char *smartd_c_cvsid="$Id: smartd.c,v 1.363 2006/04/15 14:29:25 dpgilbert Exp $" 
 ATACMDS_H_CVSID ATAPRINT_H_CVSID CONFIG_H_CVSID
 #ifdef DAEMON_WIN32_H_CVSID
 DAEMON_WIN32_H_CVSID
@@ -1578,7 +1578,7 @@ static int SCSIFilterKnown(int fd, char 
       /* <<<< This is Linux specific code to detect SATA disks using a
               SCSI-ATA command translation layer. This may be generalized
               later when the t10.org SAT project matures. >>>> */
-      req_len = 96;
+      req_len = 252;
       memset(di_buff, 0, req_len);
       if (scsiInquiryVpd(fd, 0x83, (unsigned char *)di_buff, req_len)) {
         return 0;    // guess it is normal device
